src/ui/index.js

```diff
@@ -8,6 +8,13 @@ import UiExports from './ui_exports';
 import UiBundle from './ui_bundle';
 import UiBundleCollection from './ui_bundle_collection';
 import UiBundlerEnv from './ui_bundler_env';
+import i18nPlugin from '../plugins/i18n/server/i18n/index';
+import langParser from 'accept-language-parser';
+
+let kibanaTranslations = [];
+let acceptLanguages = '';
+
+const DEFAULT_LANGUAGE = 'en';
```

Comment:
Good point, it is not explicitly defined in the design doc. Comment added to [doc](https://github.com/elastic/kibana/issues/6515#issuecomment-236924815). Will be added to the doc accordingly.